plugins/include/msgnatives.inc

```diff
@@ -0,0 +1,206 @@
+ *
+ * @noreturn
+ * @error               If "id" is non-zero and the client isn't
+ *                      connected, an error will be thrown.
+ */
+native send_geiger_signal(id, distance, bool:reliable = true)
+
+/**
+ * Hides specific elements from the HUD.
+ *
+ * @note The symbol + means that an additional crosshair will be drawn.
+ * This crosshair looks not like the regular one, but like the one that
+ * is drawn in spectator mode. You can remove this crosshair by setting
+ * the "noadd" argument to "true".
+ *
+ * @param id            Client index or 0 for all clients
+ * @param elements      HUD elements to hide. The names are self-explanatory:
+ *                        HideElement_Cross_Ammo_WPNList
+ *                        HideElement_Flashlight (+)
+ *                        HideElement_All
+ *                        HideElement_Radar_Health_Armor (+)
+ *                        HideElement_Timer (+)
+ *                        HideElement_Money (+)
+ *                        HideElement_Crosshair
+ * @param noadd         If set to false and the chosen element names have
+ *                      a "+" sign, an additional crosshair will be drawn.
+ * @param reliable      If true, the message will be sent via the reliable
+ *                      channel, otherwise it will use the unreliable one
+ *
+ * @noreturn
+ * @error               If "id" is non-zero and the client isn't
+ *                      connected, an error will be thrown.
+ */
+native hide_hud_elements(id, HideElemenentFlags:elements, bool:noadd = true, bool:reliable = true)
+
+/**
+ * Fades the client's screen.
+ *
+ * @param id            Client index or 0 for all clients
+ * @param duration      How long (in seconds) the fade is going to stay
+ *                      on the screen (0 - 16)
+ * @param fadetime      How many seconds is the fade going to fade in (0 - 16)
+ * @param flags         Screen fade flags:
+ *                        ScreenFade_FadeIn       - default
+ *                        ScreenFade_FadeOut      - fade out (not in)
+ *                        ScreenFade_Modulate     - modulate (don't blend)
+ *                        ScreenFade_StayOut      - ignores the duration and stays faded out until a new ScreenFade messages is received
+ * @param r             Red color amount (0 - 255)
+ * @param g             Green color amount (0 - 255)
+ * @param b             Blue color amount (0 - 255)
+ * @param a             Color alpha (brightness) (0 - 255)
+ * @param reliable      If true, the message will be sent via the reliable
+ *                      channel, otherwise it will use the unreliable one
+ *
+ * @noreturn
+ * @error               If "id" is non-zero and the client isn't
+ *                      connected, an error will be thrown.
+ */
+
+native fade_user_screen(id, Float:duration = 1.0, Float:fadetime = 0.5, ScreenFadeFlags:flags = ScreenFade_FadeIn, r = 0, g = 0, b = 255, a = 75, bool:reliable = true)
```

Comment:
Why default `b` is 255?

Reply:
@WPMGPRoSToTeMa Why not? It's better than having no color at all.

Reply:
@OciXCrom I think it is better to remove default values and add an example to description.